sql/maria-adele/Order_By_de_Colunas.sql

```diff
@@ -0,0 +1,5 @@
+SELECT  Rank, Name, Platform, Year, Genre, Publisher, NA_Sales, EU_Sales, JP_Sales, Other_Sales, Global_Sales
```

Comment:
Acho mais fácil usar asterisco ao invés de por o nome de todas colunas.

```suggestion
SELECT  *
```